static/bidder-info/zmaticoo.yaml

```diff
@@ -0,0 +1,10 @@
+aliasOf: "zmaticoo"
```

Comment:
This is incorrect. `zmaticoo` cannot be an alias of `zmaticoo`.

Reply:
i will change the zmaticoo.yaml to zmaticoo_ssp.yaml，is it ok?

Reply:
Please confirm if `zmaticoo` is an alias of any other adapter. If not then you need to follow normal [adapter on-boarding process](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html). 
If it is an alias then please read through the documentation on [how to create an aliased adapter.](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#aliasing-an-adapter)

Reply:
zmaticoo is new adapter，i change code follow normal [https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html](url)，Please help me check if has any  other errors?